app/src/main/ui/src/hooks/useValidationSchema.tsx

```diff
@@ -38,7 +23,7 @@ function createValidationSchema(passwordPolicy: PasswordPolicy) {
       if (!matches) {
         let translatedErrorMessage: string = t(`editUser.errors.password.${rule.Name}`);
         for (const variable of rule.Variables) {
-          translatedErrorMessage = translatedErrorMessage.replace(`{{${variable.Name}}}`, variable.Value)
+          translatedErrorMessage = t(translatedErrorMessage, {[variable.Name]: variable.Value})
```

Comment:
So you replace the replace function just with an options-object? Is variable replacement a builtin feature of translation? :o